helloworld-mutual-ssl-secured/restore-configuration.cli

```diff
@@ -4,8 +4,7 @@
 batch
 
 # Revert the changes in the undertow subsystem
-/subsystem=undertow/server=default-server/https-listener=https:undefine-attribute(name=ssl-context)
-/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=security-realm,value=ApplicationRealm)
+/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context,value=applicationSSC)
```

Comment:
`applicationSSC` is some default ssl-context that we are restoring to this value?

Reply:
ok, nevermind, I found the answer here https://docs.wildfly.org/26/WildFly_Elytron_Security.html#out-of-the-box-configuration :wink:

Reply:
actually didn't know about that section, TIL 😁